tests/test_infer.py

```diff
@@ -157,3 +160,34 @@ def test_calculator_distributions_without_teststatistic(qtilde):
     )
     with pytest.raises(RuntimeError):
         calc.distributions(1.0)
+
+
+@pytest.mark.parametrize(
+    "nsigma,expected_pval",
+    [
+        # values tabulated using ROOT.RooStats.SignificanceToPValue
+        # they are consistent with relative difference < 1e-14 with scipy.stats.norm.sf
+        (5, 2.866515718791945e-07),
+        (6, 9.865876450377018e-10),
+        (7, 1.279812543885835e-12),
+        (8, 6.220960574271829e-16),
+        (9, 1.1285884059538408e-19),
+    ],
+)
+def test_asymptotic_dist_low_pvalues(backend, nsigma, expected_pval):
+    rtol = 1e-8
+    if backend[0].float_precision != '64b':
```

Comment:
rebase to latest and this line simplifies further to

```suggestion
    if backend[0].precision != '64b':
```